src/executePrettier.test.js

```diff
@@ -77,6 +77,25 @@ describe('executePrettierOnBufferRange()', () => {
     });
   });
 
+  describe('when file is already pretty', () => {
```

Comment:
We should probably just say `when text in buffer range is already pretty` or something similar since we can format selections as well?